bot/inLineButtons_quest.py

```diff
@@ -138,9 +161,52 @@ async def handle_answer(message: types.Message, state: FSMContext, next_state: S
     if user_input not in correct_options:
         print(f"Некорректный ответ от пользователя {message.from_user.id}: {user_input}.")
         await message.answer("Пожалуйста, выберите один из предложенных вариантов ответа.")
+
+    if user_input == "/start":
+        print(f"Пользователь вышел из теста {message.from_user.id}: {user_input}.")
+        await set_main_menu(bot)
+        await state.clear()
+        await message.answer("Вы вышли из прохождения теста командой старт\nОцените продуманность кода по 10 бальной шкале\nНапишите цифру от 1 до 10.")
```

Comment:
I don't see where user's answer is being saved? And I tested this and message with rate was not handled
```console
2024-09-29 21:17:13,873 : INFO : aiogram.event : Update id=427536492 is not handled. Duration 1 ms by bot id=7819575371
```

Reply:
I have everything displayed, I will attach screenshots
<img width="1027" alt="image" src="https://github.com/user-attachments/assets/4ae3c816-96e8-4c9b-9b59-d81ca874860d">
<img width="574" alt="image" src="https://github.com/user-attachments/assets/644b239d-c46e-4e20-9704-d402f65fa94d">

Reply:
Yeah. Here you are right, but if you leave a number as a rate for our code
Here you can see (especially in time of message being sent and its log time) After rate – bot doesn't answer and just being quiet
<img width="1439" alt="image" src="https://github.com/user-attachments/assets/9e0f8eea-e7c6-4862-88b7-f011cf4ac88e">

Reply:
I understand you, I will add processing of user ratings in the next update

Reply:
Currently I will comment this feature for later work. But this can be a good concept for future feedback collection! Thanks!